### ISSUE TITLE
BVH too little

### ISSUE THREAD

**Issue by ejuoy** [facebookresearch/AnimatedDrawings#297]
Can you open all the bvh on your display web page?

**Comment by TejasKumar3005** [comment]
I agree, @hjessmith. Please share all the BVH files on the web page; they are well suited for animating drawings, while most of the BVH available on the internet are not suited for animated drawings.

**Comment by hjessmith** [comment]
Sorry folks, many of them were taken from Adobe Mixamo, and it's against their ToS to redistribute the motion data directly. 

You can download them yourself and use Maya or Blender to convert to BVH, though.
